assets/ui/blogger-sans-medium-30.tres

```diff
@@ -3,5 +3,6 @@
 [ext_resource path="res://assets/ui/blogger-sans-medium.otf" type="DynamicFontData" id=1]
 
 [resource]
-size = 48
+size = 36
```

Comment:
This size does not match the filename.

Reply:
Is this resource needed? It looks like it's identical to `blogger-sans-medium-36.tres`

Reply:
Changed fonts. Removed `blogger-sans-medium-30.tres` and introduced a new `blogger-sans-bold-36.tres` used for the nametag.